go/roothash/api/api.go

```diff
@@ -162,15 +162,23 @@ type ExecutionDiscrepancyDetectedEvent struct {
 type MergeDiscrepancyDetectedEvent struct {
 }
 
+// FinalizedEvent is a finalized event.
+type FinalizedEvent struct {
+	Value []byte `json:"value"`
```

Comment:
Why is this an opaque value? Should this just have the round number?

Reply:
Agreed, fixed